src/app/components/products-sidebar/products-sidebar.component.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { CategoryModel } from 'src/app/models/category.model';
+
+@Component({
+  selector: 'app-products-sidebar',
+  styleUrls: ['./products-sidebar.component.scss'],
+  templateUrl: './products-sidebar.component.html',
+  encapsulation: ViewEncapsulation.Emulated,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProductsSidebarComponent {
+  @Input() categories$: Observable<CategoryModel[]> = of([]);
```

Comment:
you should not pass observables as inputs, you can use | async pipe in parent component and here you get only categoryModel - also be aware of passing entire models like this. Presentation components (with @Input) should define their own query models